gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -546,7 +549,29 @@ def UpdateSlot (pc, slot):
 
 	Window = GemRB.GetView("WIN_INV")
 
-	SlotType = GemRB.GetSlotType (slot+1, pc)
+	if not GameCheck.IsPST:
```

Comment:
```suggestion
	if not GameCheck.IsPST ():
```

Reply:
Also, does python not care about scope here (slot_item, SlotType, using_fists)?

Reply:
That's just me being really dumb, fixed over here locally